lsp-pyls.el

```diff
@@ -350,6 +356,7 @@ parameters referenced in config."
 (lsp-register-custom-settings
  '(("pyls.rope.ropeFolder" lsp-pyls-rope-rope-folder)
    ("pyls.rope.extensionModules" lsp-pyls-rope-extension-modules)
+   ("pyls.plugins.autopep8.enabled" lsp-pyls-plugins-autopep8-enabled t)
    ("pyls.plugins.yapf.enabled" lsp-pyls-plugins-yapf-enabled t)
```

Comment:
And here

Reply:
Isn't the `t` just to tell lsp that the variable is of type boolean!?

Reply:
You are correct, I am sorry